datalake/src/main/java/com/sequenceiq/datalake/flow/start/event/SdxStartStartEvent.java

```diff
@@ -1,16 +1,25 @@
 package com.sequenceiq.datalake.flow.start.event;
 
+import static com.sequenceiq.datalake.flow.start.SdxStartEvent.SDX_START_EVENT;
+
+import com.sequenceiq.cloudbreak.common.event.AcceptResult;
 import com.sequenceiq.datalake.flow.SdxEvent;
 
+import reactor.rx.Promise;
+
 public class SdxStartStartEvent extends SdxEvent {
 
     public SdxStartStartEvent(String selector, Long sdxId, String userId) {
         super(selector, sdxId, userId);
     }
 
+    public SdxStartStartEvent(String selector, Long sdxId, String userId, Promise<AcceptResult> accepted) {
```

Comment:
if the `selector()` is overwritten, it doesn't make any sense to expect a `selector` parameter

Reply:
Removed the override.